lib/mix/tasks/talon.gen.theme.ex

```diff
@@ -220,6 +223,41 @@ defmodule Mix.Tasks.Talon.Gen.Theme do
   end
   defp gen_generators(config), do: config
 
+  def gen_dashboard_template(%{dashboard: true} = config) do
+    binding = Kernel.binding() ++
+      [title_txt: gettext("Dashboard"),
```

Comment:
Okay, I was wonder exactly how we wanted to do that. I was looking at what Jose was saying here: https://github.com/smpallen99/coherence/issues/185